File: 27-remove-element/27-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        l,idx=len(nums),0
        while idx<l:
            if nums[idx]==val:
                nums[idx]=nums[l-1]
                l-=1
            else:
                idx+=1
        return l
                

File: 27-remove-element/27-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int length=nums.size(),idx=0;
        while(idx<length){
            if(nums[idx]==val){
                nums[idx]=nums[length-1];
                length--;
            }
            else{
                idx++;
            }
        }
        return length;
    }
};